deployment/terraform/create.go

```diff
@@ -181,8 +220,34 @@ func (t *Terraform) setupAppServers(output *terraformOutput, extAgent *ssh.ExtAg
 	}
 }
 
+func (t *Terraform) setupLoadtestAgents(extAgent *ssh.ExtAgent, output *terraformOutput) error {
+	// Creating the sysadmin user.
+	// should wait the server to start
+
+	for i := 0; i < len(output.Agents.Value); i++ {
```

Comment:
Nit: any reason we are not using a range here ?

Reply:
Hmm, no reason. It was starting from 1 in the earlier version of this PR. Changed to range 👍